test/src/com/xilinx/rapidwright/design/TestDesignTools.java

```diff
@@ -979,4 +980,31 @@ public void testMakePhysNetNamesConsistentLogicalVccGnd() {
         }
         Assertions.assertEquals(31, numSitewires);
     }
+
+    @Test
+    public void testPlaceCell() {
+        //test a design that already contains a Carry4 cell
+        Design d0 = Design.readCheckpoint("bug709.dcp");
```

Comment:
```suggestion
        Design d0 = RapidWrightDCP.loadDCP("bug709.dcp");
```
The test is failing because it can't find the right DCP, this should fix it.